docs/ocm-control-plane-walkthrough.md

```diff
@@ -108,39 +116,39 @@ EOF
 
 ### Create the gateway class
  
-Next we will set up our multi-cluster gateway class
+Lastly, we will set up our multi-cluster gateway class. In `T1`, run:
 
-```
+```bash
 kubectl create -f hack/ocm/gatewayclass.yaml
 ```
 
-### Start the gateway controller
+### Start the Gateway Controller
 
-in the terminal with the hub cluster context and from the root of the repo run:
+In `T2` run the following to start the Gateway Controller:
 
-```
+```bash
 (export $(cat ./controller-config.env | xargs) && export $(cat ./aws-credentials.env | xargs) && make build-controller install run-controller)
-
 ```
 
 ### Check the managed zone
 
-Lets ensure our managedzone is present. Open a new terminal: 
+Lets ensure our `managedzone` is present. In `T1`, run the following:
 
-```
+```bash
 export KUBECONFIG=$(pwd)/local/kube/control-plane.yaml
 kubectl get managedzone -n multi-cluster-gateways
```

Comment:
Small suggestion here for something that is a little annoying.
Split the cmd and the output into separate blocks.

Reply:
Hard agree - I'll do these here and for the others.